src/index.tsx

```diff
@@ -133,7 +134,7 @@ const MuiFileInput = <T extends boolean | undefined>(
       InputProps={{
         startAdornment: (
           <InputAdornment position="start">
-            <AttachFileIcon />
+            {!hideIcon ? <AttachFileIcon /> : null}
```

Comment:
why keep the `InputAdornment` ?

Reply:
If hideIcon is enabled, `InputAdornment` is not necessary as it seems I unconsciously followed the `hideSizeText` code.

Reply:
```tsx
startAdornment: !hideIcon ? (
	<InputAdornment position="start">
		<AttachFileIcon />
	</InputAdornment>
) : null,
```
If you make changes to the code as above,

![image](https://github.com/viclafouch/mui-file-input/assets/75025529/f24b4c5b-2075-45eb-8f46-ebdb07f76193)

The label overlaps with the placeholder as shown in the image

```tsx
startAdornment: !hideIcon ? (
	<InputAdornment position="start">
		<AttachFileIcon />
	</InputAdornment>
) : (
	<></>
),
```
Therefore, I'm planning to modify the code as shown above. @viclafouch  What do you think?

Reply:
LGTM

Reply:
@viclafouch  I have done applying it

Reply:
But I think ESlint throw an error here because of an empty React Fragment, no ?